resources.go

```diff
@@ -1219,11 +1234,33 @@ func Provider() tfbridge.ProviderInfo {
 			"azurerm_healthcare_service": {Tok: azureResource(azureHealthcare, "Service")},
 
 			// NetApp
-			"azurerm_netapp_account": {Tok: azureResource(azureNetapp, "Account")},
-			"azurerm_netapp_pool":    {Tok: azureResource(azureNetapp, "Pool")},
+			"azurerm_netapp_account":  {Tok: azureResource(azureNetapp, "Account")},
+			"azurerm_netapp_pool":     {Tok: azureResource(azureNetapp, "Pool")},
+			"azurerm_netapp_volume":   {Tok: azureResource(azureNetapp, "Volume")},
+			"azurerm_netapp_snapshot": {Tok: azureResource(azureNetapp, "Snapshot")},
 
 			//AppConfiguration
 			"azurerm_app_configuration": {Tok: azureResource(azureAppConfiguration, "ConfigurationStore")},
+
+			// Backup
+			"azurerm_backup_container_storage_account": {Tok: azureResource(azureBackup, "ContainerStorageAccount")},
+			"azurerm_backup_policy_file_share":         {Tok: azureResource(azureBackup, "PolicyFileshare")},
+			"azurerm_backup_protected_file_share":      {Tok: azureResource(azureBackup, "ProtectedFileShare")},
+			"azurerm_backup_policy_vm":                 {Tok: azureResource(azureBackup, "PolicyVm")},
+			"azurerm_backup_protected_vm":              {Tok: azureResource(azureBackup, "ProtectedVm")},
```

Comment:
I think "Vm" should be "VM" everywhere (let me know if we already had examples of otherwise)

Reply:
fixed